File: daq/slc/extra/javalib/src/b2daq/dqm/core/HorizontalRange.java
package b2daq.dqm.core;

public class HorizontalRange extends Range {

	public HorizontalRange(String name, double min, double max) {
		super(name, min, max);
	}

	public HorizontalRange(String name, double min) {
		super(name, min, min);
	}

	public HorizontalRange() {
		super("", 0, 0);
	}

	@Override
	public String getDataType() {
		return "R1H";
	}

}

File: daq/slc/dqm/src/HorizontalRange.cc
#include "daq/slc/dqm/HorizontalRange.h"

using namespace Belle2;

HorizontalRange::HorizontalRange() : Range()
{

}

HorizontalRange::HorizontalRange(const std::string& name) : Range(name)
{

}

HorizontalRange::HorizontalRange(const std::string& name, double min, double max)
  : Range(name, min, max)
{

}

HorizontalRange::~HorizontalRange() throw()
{

}

std::string HorizontalRange::getDataType() const throw()
{
  return "R1H";
}
